### ISSUE TITLE
Search icon in search input lacks 3:1 contrast ratio

### ISSUE THREAD

**Issue by shainanigans** [StackExchange/Stacks#997]
**Describe the bug**
The search icon in the [search input](https://stackoverflow.design/product/components/inputs/#search) lacks the 3:1 color contrast ratio required for non-text elements that convey meaning.

**To Reproduce**
1. Go to documentation > Inputs > Search
2. See the color contrast of the search icon against the input background in both light and auto dark mode.

**Expected behavior**
The color contrast for the icon should be at least 3:1 against its background (changing the icon color to `black-400` should work for both light and auto dark mode).

**Screenshots**
<img width="151" alt="Screen Shot 2022-06-27 at 6 01 39 PM" src="https://user-images.githubusercontent.com/14816224/176043201-54d5b296-851d-42a0-bf8d-466040f54000.png">
<img width="137" alt="Screen Shot 2022-06-27 at 6 01 45 PM" src="https://user-images.githubusercontent.com/14816224/176043203-4c57dd6b-424e-4cbc-89cd-02b839ece5a3.png">

**Additional context**
This is not an issue in the forced dark mode of the Teams topbar.
